--- pcrro8a.t1asm
+++ fcrro8a.t1asm
@@ -4601,15 +4601,15 @@
 	endchar
 	} |-
 /quoteleft {
-	228 600 hsbw
+	409 600 hsbw
 	0 233 vstem
 	392 callsubr
 	17 4 callsubr
-	104 320 rmoveto
+	129 320 rmoveto
 	dotsection
-	129 243 rlineto
-	-73 hlineto
-	-160 -243 rlineto
+	-25 243 rlineto
+	-104 hlineto
+	56 -243 rlineto
 	403 callsubr
 	} |-
 /a {
@@ -5597,23 +5597,23 @@
 	403 callsubr
 	} |-
 /quotedblleft {
-	213 600 hsbw
+	322 600 hsbw
 	0 404 vstem
 	392 callsubr
 	102 4 callsubr
-	276 320 rmoveto
+	302 320 rmoveto
 	dotsection
-	128 243 rlineto
-	-72 hlineto
-	-160 -243 rlineto
+	-26 243 rlineto
+	-104 hlineto
+	58 -243 rlineto
 	closepath
 	dotsection
 	103 4 callsubr
-	-68 hmoveto
+	-100 hmoveto
 	dotsection
-	128 243 rlineto
-	-72 hlineto
-	-160 -243 rlineto
+	-26 243 rlineto
+	-104 hlineto
+	58 -243 rlineto
 	403 callsubr
 	} |-
 /guillemotleft {
